lib/config/options/index.ts

```diff
@@ -256,9 +256,8 @@ const options: RenovateOptions[] = [
   {
     name: 'printConfig',
     description:
-      'If enabled, Renovate logs the fullly resolved config for each repo, plus the fully resolved presets.',
+      'If enabled, Renovate logs the fully resolved config for each repo, plus the fully resolved presets.',
     type: 'boolean',
-    globalOnly: true,
```

Comment:
Do we discussed and aggreed this change? 🤔

Reply:
I think we can make this allowed for repos